build-ps/percona-server.spec

```diff
@@ -495,10 +500,15 @@ datadir=$(/usr/bin/my_print_defaults server mysqld | grep '^--datadir=' | sed -n
 %else
 /sbin/chkconfig --add mysql
 %endif
-# We need this because we don't provide my.cnf on centos 6
-# and the default system one doesn't have info needed
-%if 0%{?rhel} < 7
-if [ $1 -eq 1 ]; then
+if [ ! -f /etc/my.cnf ]
+then
+  update-alternatives --install /etc/my.cnf my.cnf "/etc/percona-server.cnf" 200
+else
+  echo " -------------"
+  echo "   *  The suggested mysql options and settings are in /etc/percona-server.conf.d/mysqld.cnf"
+  echo "   *  If you want to use mysqld.cnf as default configuration file please use the following command:"
```

Comment:
The problem with this is that if I already have a my.cnf and issue that command nothing happens - I guess that's why the --force option on debian/ubuntu. The users might get confused by this so maybe we should add something like "If you want to use mysqld.cnf as default configuration file please remove the existing my.cnf and use the following command:"